### PR TITLE
Fix#14986 Removed Quotes from SQL statement and unecessary zeroes after decimal

### LINKED ISSUE #14986: #1292 error for insert format fractional current_timestamp, support custom default functions
phpMyAdmin version 4.8.5
MariaDB 10.3.12
php 7.3.1

create table log:
```sql
CREATE TABLE IF NOT EXISTS log (
  logtime datetime(2) NOT NULL DEFAULT current_timestamp(2),
  logtext varchar(255) NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
```

select the table and click insert

the logtime value is prompted as `current_timestamp(2).000000`

1) this result in a `#1292 - Incurrent datetime value: 'current_timestamp(2).00000' for column logtime`
2) removing the trailing .000000 also results in the same error

A working statement would be `insert into log values( default, 'test' )`
[ts-test.gz](https://github.com/phpmyadmin/phpmyadmin/files/2918629/ts-test.gz)

### PR DESCRIPTION
**Pull request by kartik1000**
Signed-off-by: Kartik Kathuria <kathuriakartik0@gmail.com>

### Description

This PR will ensure that the `Current_Timestamp()` based SQL Query executes correctly. It has also removed the invalid `.00000` zeroes which were unnecessary based on current execution of query and provided invalid value to SQL Query. The changes have been implemented based on the discussion in #14986 and on the fact that the maximum argument of Current_Timestamp is 6 as it is its maximum precision. 

Fixes #14986 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/InsertEdit.php [hunk 1]
```diff
@@ -2752,8 +2752,8 @@ public function getCurrentValueForDifferentTypes(
                 $current_value = "b'" . $this->dbi->escapeString($current_value)
                     . "'";
             } elseif (! ($type == 'datetime' || $type == 'timestamp')
-                || ($current_value != 'CURRENT_TIMESTAMP'
-                    && $current_value != 'current_timestamp()')
+                || ($current_value != 'CURRENT_TIMESTAMP' && $current_value != 'CURRENT_TIMESTAMP()' && $current_value != 'CURRENT_TIMESTAMP(1)' && $current_value != 'CURRENT_TIMESTAMP(2)' && $current_value != 'CURRENT_TIMESTAMP(3)' && $current_value != 'CURRENT_TIMESTAMP(4)' && $current_value != 'CURRENT_TIMESTAMP(5)' && $current_value != 'CURRENT_TIMESTAMP(6)'
```
Comment (williamdes) [inline libraries/classes/InsertEdit.php]: Can you use `in_array`   ?

Comment (kartik1000) [reply libraries/classes/InsertEdit.php]: Yeah Sure :)

**Comment by williamdes** [comment]
I think it is going to be more complicated

https://stackoverflow.com/a/52769071/5155484

```sql
CREATE TABLE test ( uuid BINARY(16) PRIMARY KEY DEFAULT unhex(replace(uuid(),'-','')) ); 
INSERT INTO test () VALUES ();
SELECT * FROM test;
```

```sql
CREATE TABLE testyear ( uuid int(11) PRIMARY KEY DEFAULT year(now() )) ; 
```

**Comment by AghaSaad04** [comment]
@kartik1000, This solution is not working on my side, I am using the demo server.

**Comment by kartik1000** [comment]
@sudo-bot Deploy pr

**Comment by kartik1000** [comment]
@sudo-bot Deploy PR

**Pull request closed without merging**